js-packages/@prestojs/rest/src/dedupeInFlightRequestsMiddleware.ts

```diff
@@ -0,0 +1,73 @@
+import { EndpointRequestInit, MiddlewareFunction } from './Endpoint';
+
+function defaultGetKey(url: string, requestInit: EndpointRequestInit): string {
+    return [
+        url,
+        requestInit.method,
+        [...requestInit.headers.entries()].map(([key, value]) => `${key}⁞${value}`).join('__'),
```

Comment:
See above; this will treat the order of headers as significant

(edge case: I don't think it's valid to have duplicate headers in HTTP is it?)

Reply:
the headers object can't have duplicates no